Bio/SwissProt/__init__.py

```diff
@@ -136,10 +136,18 @@ def read(handle):
         raise ValueError("More than one SwissProt record found")
     return record
 
+def strip_evidence(value):
+    """Strip evidence codes from values.
+    ORFNames=N310_11497 {ECO:0000313|EMBL:KFP74115.1}; --> ORFNames=N310_11497;
+    SubName: Full=Tumor protein 63 {ECO:0000313|EMBL:KFP74115.1}; --> SubName: Full=Tumor protein 63;
+    """
+    # this should work also for lines with multiple entries / evidence codes
```

Comment:
I have also noticed in some entries, the GN is split over multiple lines ie. http://www.uniprot.org/uniprot/A0A0D6JXN9.txt

Plus, entry from the website differs slightly from that from dump: in the website it's split after coma, while in the dump after `BN996_03901`! Because of that, I think it makes sense to remove evidence codes after all lines are loaded. 
I'm testing it on swissprot and trembl dump. 

```
GN   ORFNames=BN996_03895 {ECO:0000313|EMBL:CQR54010.1},
GN   BN996_03901 {ECO:0000313|EMBL:CQR54021.1};
```

```bash
wget -O- ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_trembl.dat.gz | zgrep -m1 -C10 'BN996_03895'

...
GN   ORFNames=BN996_03895 {ECO:0000313|EMBL:CQR54010.1}, BN996_03901
GN   {ECO:0000313|EMBL:CQR54021.1};
...
```